ArduSub/Parameters.cpp

```diff
@@ -767,6 +767,13 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("ORIGIN_ALT", 21, ParametersG2, backup_origin_alt, 0),
 
+    // @Param: SURF_NOBARO_THST
+    // @DisplayName: Throttle output when surface mode is engaged and baro is not available
```

Comment:
not a blocker of course but normally with Parameter descriptions I try to put the most important word first.  So in this case, I'd start it with "Surface mode".  So for example it could be, "Surface mode throttle output when no barometer is available"

Reply:
thanks! updated